spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -1699,6 +1703,7 @@ def _do_load(self, graph_changed):
             self._config.getboolean("Reports", "write_memory_map_report") and
             graph_changed
         )
+        inputs["NoSyncChanges"] = self._no_sync_changes
```

Comment:
It isn't clear why this is needed

Reply:
i think this is a slip from having bitfield stuff inside the ear branches. verification might be to chekc if this is in all_algorithms.

Reply:
Yes, looks like it - removing!